From 5b7ece13c9fcb7559c7e792a04789a1d0a7468b2 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Mon, 9 Dec 2024 17:31:56 +0800
Subject: [PATCH 221/301] AOSCOS: gpio: gpio-nct6102: drop an unused variable
 in nct6102_gpio_setup()

The integer-type variable `ret' was never used in the function, causing an
error with `-Werror=unused-variable'.

Remove this unused variable.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 drivers/gpio/gpio-nct6102.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpio/gpio-nct6102.c b/drivers/gpio/gpio-nct6102.c
index c376f9fd6d07..1bb29ea5f5f2 100644
--- a/drivers/gpio/gpio-nct6102.c
+++ b/drivers/gpio/gpio-nct6102.c
@@ -170,7 +170,6 @@ static struct gpio_chip nct6102_chip = {
 
 static int __init nct6102_gpio_setup(void)
 {
-	int ret;
 	u16 dev_id;
 
 	/* Detect device */
-- 
2.48.1

